deploy/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -393,6 +393,36 @@ rules:
 ---
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
+metadata:
```

Comment:
For the helm chart, let's only create the rbac or other nfs config if the nfs driver is installing

Reply:
done